### PR TITLE
Document elements with overflow hidden are considered non-scrollable (close #144)

### LINKED ISSUE #144: Document elements with 'overflow=hidden' should be considered non-scrollable
T303226

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

File: src/client/core/utils/style.js [hunk 1]
```diff
@@ -80,7 +80,10 @@ export function hasScroll (el, doc) {
     var isHtmlElement   = /html/i.test(el.tagName);
     var body            = isHtmlElement ? $(el).find('body')[0] : null;
 
-    if (!overflowX && !overflowY && !isHtmlElement)
+    //T303226
+    if (isHtmlElement && styles.overflowX === 'hidden' && styles.overflowY === 'hidden')
+        return false;
+    else if (!isHtmlElement && !overflowX && !overflowY)
```
Comment (inikulin) [inline src/client/core/utils/style.js]: No need for `else` here, since we have `return` in previous condition

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**